plugin/adsk/scripts/AETemplateHelpers.py

```diff
@@ -11,6 +12,46 @@ def debugMessage(msg):
     if DEBUG:
         print(msg)
 
+def GetAllRootPrimNamesNaturalOrder(proxyShape):
+    # Custom comparator
```

Comment:
The method below is a customized comparator for sorting in natural order. For example, proxyShape10 will come after proxyShape 2.